script.sh

```diff
@@ -0,0 +1,11 @@
+VM_ID=$1
+IMAGE_PATH=$2
+STORAGE_POOL=$3
+# The new imported disk will get a name that depends on the current number of disks for the vm 
+DISK_COUNT=$4
+
+qm stop $VM_ID
+qm importdisk $VM_ID $IMAGE_PATH $STORAGE_POOL
+qm set $VM_ID --scsi0 $STORAGE_POOL:vm-$VM_ID-disk-$DISK_COUNT
+qm set $VM_ID --boot order=scsi0
+qm start $VM_ID
```

Comment:
Fix [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086):
```suggestion
qm stop "$VM_ID"
qm importdisk "$VM_ID" "$IMAGE_PATH $STORAGE_POOL"
qm set "$VM_ID" --scsi0 "$STORAGE_POOL:vm-$VM_ID-disk-$DISK_COUNT"
qm set "$VM_ID" --boot order=scsi0
qm start "$VM_ID"
```